free-programming-books-es.md

```diff
@@ -241,6 +241,7 @@
 #### React
 
 * [Desarrollo de Aplicaciones Web con React.js y Redux.js](https://leanpub.com/react-redux) (PDF, EPUB, MOBI)
+* [El Camino para Aprender React](https://leanpub.com/the-road-to-learn-react-spanish) (PDF, EPUB, MOBI)
```

Comment:
On the English list Leanpub books using this model have an added notation "(Leanpub account or valid email requested)". Could you add  a spanish version of this to your addition? (Separately, we can add the same notation to other Leanpub books on the spanish list.)

Also, it helps to identify books, especially translated works, to add the author name to the listing.